src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsRecoveryObject.java

```diff
@@ -48,46 +38,45 @@ public void verifyPageFairRecoveryWithAdBlock() {
     }
 
     // when PF recovered ad is on page, inserts span elements as a direct children of body
-    wait.forElementPresent(spansBodyChildrenSelector);
+    wait.forElementPresent(PF_RECOVERED_ADS_SELECTOR);
 
     // verify that adblock is turned on on that page
     verifyNoAdsOnPage();
 
-    String firstSpanClass = driver.findElement(spansBodyChildrenSelector).getAttribute("class");
-    List<WebElement> recoveredAds = driver
-        .findElements(By.cssSelector("body>span." + firstSpanClass))
-        .stream()
-        .filter(WebElement::isDisplayed)
-        .filter(e -> e.getCssValue("background").contains("data:image/jpeg"))
-        .collect(Collectors.toList());
+    List<WebElement> recoveredAds = getRecoveredAds(PF_RECOVERED_ADS_SELECTOR);
 
     Assert.assertEquals(recoveredAds.size(), RECOVERABLE_ADS_COUNT);
 
     for (WebElement ad : recoveredAds) {
       Dimension adSize = ad.getSize();
 
-      if (adSize.equals(topLeaderboardSize)) {
+      if (adSize.equals(TOP_LEADERBOARD_SIZE)) {
         Assertion.assertTrue(ad.getCssValue("background").contains(expectedRecoveredLB), "TOP_LEADERBOARD is not correctly recovered!");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "background" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)